dll/cpl/desk/lang/bg-BG.rc

```diff
@@ -179,14 +179,14 @@ CAPTION "Общи"
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Екран", -1, 7, 6, 237, 56, WS_GROUP
-    LTEXT "Размер на шрифта:", -1, 14, 20, 222, 9
+    LTEXT "Размер на шрифта:", -1, 14, 20, 222, 9//FIXME: add accel
     COMBOBOX IDC_FONTSIZE_COMBO, 14, 30, 223, 80, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 46, 223, 12
     GROUPBOX "Съвместимост", -1, 7, 67, 237, 97
     LTEXT "Някои приложения няма да работят правилно, ако не презапуснете компютъра, след като промените цветовите настройки.\nСлед като променя цветовите настройки", -1, 12, 78, 228, 34
-    AUTORADIOBUTTON "&Презапуск преди прилагане на новите цветови настройки.", IDC_RESTART_RB, 14, 116, 228, 10
-    AUTORADIOBUTTON "Прилагане на новите настройки без презапуск.", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
-    AUTORADIOBUTTON "Питане преди прилагане на новите настройки", IDC_ASKME_RB, 14, 146, 228, 10
+    AUTORADIOBUTTON "&Презапуск преди прилагане на новите цветови настройки", IDC_RESTART_RB, 14, 116, 228, 10
+    AUTORADIOBUTTON "Прилагане на новите настройки без презапуск", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10//FIXME: add accel
+    AUTORADIOBUTTON "Питане преди прилагане на новите настройки", IDC_ASKME_RB, 14, 146, 228, 10//FIXME: add accel
```

Comment:
It looks like similar accelerators as those from russian _could_ be used there as well (just to note...)

Reply:
I guessed for the backports, and left master head to be supplied by the native speakers.